components/homepage/components/Creative/index.js

```diff
@@ -0,0 +1,94 @@
+import { Swiper, SwiperSlide } from "swiper/react";
+import SwiperCore, { Pagination } from "swiper";
+import "swiper/css";
+import "swiper/css/pagination";
+import {
+  StCreativeContainer,
+  StCreativeTitle,
+  StCreativeContant,
+  StCreativeContantLinks,
+  StCreativeContantInfo,
+  StCreativeContantSingleLink,
+  StCreativeFadeOne,
+  StCreativeFadeTwo,
+  StCreativeContantInfoBtn,
+  StCreativeContantImage,
+  StCreativeContantImageSwiper,
+} from "./styles";
+
+const LIST_ITEMS = [
+  "the objective of the campaign",
+  "hom much to spend",
+  "who to target",
+  "the type of advertising you'll see",
+];
+
+const index = () => {
+  SwiperCore.use(Pagination);
+  return (
+    <StCreativeContainer>
+      <StCreativeTitle>
+        <span>CREATIVE</span>
+        <h3>Know your ads inside and out-perform</h3>
+      </StCreativeTitle>
+      <StCreativeContant>
+        <StCreativeContantLinks>
+          {LIST_ITEMS.map((item, index) => (
+            <StCreativeContantSingleLink key={index}>
+              <img src="./images/pencile.png" alt="pencile" />
+              {item}
+            </StCreativeContantSingleLink>
+          ))}
+          <StCreativeFadeOne />
+          <StCreativeFadeTwo />
+        </StCreativeContantLinks>
+        <StCreativeContantInfo>
+          <h4>Every secound counts , so Every details matters</h4>
```

Comment:
At texts, every where, you should remove space before `,` ,`.`,` (` `)`, `?` and `!`. It is wrong to add space before these symbols